lib/daru/plotting/nyaplot/dataframe.rb

```diff
@@ -21,6 +21,11 @@ module NyaplotLibrary
         #   df = Daru::DataFrame.new({a:['A', 'B', 'C', 'D', 'E'], b:[10,20,30,40,50]})
         #   df.plot type: :bar, x: :a, y: :b
         def plot opts={}, &block
+          unless opts[:x]
+            opts[:x] = :_index
+            self.[]=(:_index, @index.to_a)
```

Comment:
Why not `self[:_index] = @index.to_a`?

Reply:
No specific reason. I made the change :)